.github/workflows/_buck-build-test.yml

```diff
@@ -21,11 +21,10 @@ jobs:
           distribution: 'temurin'
 
       - name: Setup miniconda
-        uses: conda-incubator/setup-miniconda@v2
+        uses: pytorch/test-infra/.github/actions/setup-miniconda@main
         with:
-          auto-update-conda: true
           python-version: 3.8
-          activate-environment: build
+          environment-file: .github/requirements/conda-env-ubuntu-X64
```

Comment:
I think it's better to use the syntax `.github/requirements/conda-env-${{ runner.os }}-${{ runner.arch }}` here.  From what I see in https://github.com/pytorch/pytorch/actions/runs/3516771558/jobs/5893753273. The file name would become  `conda-env-Linux-X64` instead

Reply:
Implemented the changes.